src/librustc_lint/builtin.rs

```diff
@@ -41,6 +41,7 @@ use lint::{Level, Context, LintPass, LintArray, Lint};
 
 use std::collections::BitSet;
 use std::collections::hash_map::Entry::{Occupied, Vacant};
+#[allow(deprecated)]
 use std::num::SignedInt;
```

Comment:
Yes, I tried to remove the ones that were only used for methods, but I may have missed some...